dashboard/lib/policies/child_account.rb

```diff
@@ -1,21 +1,27 @@
+require 'cdo/shared_constants'
+require 'cpa'
+require 'date'
+
 class Policies::ChildAccount
   # Values for the `child_account_compliance_state` attribute
   module ComplianceState
     # The student's account has been used to issue a request to a parent.
-    LOCKED_OUT = 'l'.freeze
+    LOCKED_OUT = SharedConstants::CHILD_ACCOUNT_COMPLIANCE_STATES.LOCKED_OUT
```

Comment:
Is it a little odd to be setting a constant to the value of another constant? Should we delete `ComplianceState` and have `SharedConstants` be the source of truth? If you do this refactor, make sure to move the documentation for the constants over to `SharedConstants`